app/main.py

```diff
@@ -1 +1,51 @@
-#  write your code here
+from __future__ import annotations
```

Comment:
The `from __future__ import annotations` statement is not necessary here unless you are planning to use postponed evaluation of type annotations for forward references. If not needed, you can remove this line.